icepyx/__init__.py

```diff
@@ -1,3 +1,15 @@
+from warnings import warn
+
+deprecation_msg = """icepyx v1.x is deprecated; the back-end systems on which is relies
+are being shut down. Please upgrade to icepyx v2.x which uses the new NASA Harmony
+back-end.
```

Comment:
Could we say "is going to be deprecated"? As written, this suggests v2 is already available for upgrading to...
```suggestion
deprecation_msg = """icepyx v1.x is going to be deprecated; the back-end systems on which it relies
are being shut down. Please upgrade to icepyx v2.x which uses the new NASA Harmony
back-end.
```

Reply:
I agree we should clarify that the 2.x upgrade is not yet available.

However, "is going to be deprecated" is IMO confusing terminology. "Deprecated" colloquially means "scheduled for decommissioning or removal" ([see Python's "deprecations" page for examples](https://docs.python.org/3/deprecations/index.html); all of the things there are already considered "deprecated", but will be removed in future releases). So I would argue that the message "1.x is deprecated" is correct, but the wording around it could be clearer.

Reply:
What do you think? 08db71d